glossary.html

```diff
@@ -81,7 +82,7 @@ <h1>
         </a>
         2019
         </p>
-        <img src="sc-logo.png" sizes="70px" srcset="sc-logo@1.5x.png 105w", "sc-logo@2x.png 140w", "sc-logo@3x.png 210w", "sc-logo@4x.png 280x">
+        <img src="images/sc-logo.png" sizes="70px" srcset="images/sc-logo@1.5x.png 105w, images/sc-logo@2x.png 140w, images/sc-logo@3x.png 210w, images/sc-logo@4x.png 280w" alt="Sault College Logo">
```

Comment:
A few issues here.  
1) You need the full relative URL (include the images folder)
2) The comma separated list is inside of a SINGLE set of quotation marks (it's all part of one value for the srcset attribute)
3) Wrong unit on the last measurement